frontend/src/App.css

```diff
@@ -1,4 +1,18 @@
 .add-item-list {
     border: none;
     border-bottom: 1px solid black;
+}
+
+.flash {
+    position: absolute;
+    z-index: 1;
+}
+
+.app {
+    position: relative;
```

Comment:
what's the effect of position relative here? Why is it added?

Reply:
oops, old style. removed.